devicetypes/smartthings/smartsense-button.src/smartsense-button.groovy

```diff
@@ -272,7 +272,8 @@ def refresh() {
 def configure() {
     // Device-Watch allows 2 check-in misses from device + ping (plus 1 min lag time)
     // enrolls with default periodic reporting until newer 5 min interval is confirmed
-    sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 1 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
```

Comment:
The `checkInterval` event is also created in `parse`. I think it should be safe to remove that code and just set the final desired checkInterval value here in the `DeviceWatch-Enroll`. Anyone feel that isn't a good idea?

Reply:
That (much shorter) checkInterval is a response to the config reporting response. i.e. have a very long check interval until the device confirms it will be reporting to us regularly.